lib/league_statable.rb

```diff
@@ -11,6 +11,7 @@ def average_goals_per_team_id
     goals_made.each do |key, value|
       average_goals_per_team[key] = (value / games_played_per_team[key]).round(4)
     end
+    average_goals_per_team
```

Comment:
nice looks like that was an easy fix! I wonder how we didn't catch that the first time, but as long as it works now!